avae/evaluate.py

```diff
@@ -169,17 +169,30 @@ def evaluate(
     # visualise latent disentanglement
     if settings.VIS_DIS:
         vis.latent_disentamglement_plot(
-            x_test, vae, device, data_dim, poses=p_test, mode="_eval"
+            batch[0].shape[-data_dim:],
+            x_test,
+            vae,
+            device,
+            data_dim,
```

Comment:
would dsize and data_dim redundant?

Reply:
no no, it is the data dimension so a list of for example [64,64,64] , you comment is a good suggestion , implementing now